### DISCUSSION TITLE
Security Onion 2.0 in AWS

### DISCUSSION THREAD

**Discussion by hirenshah005** [Security-Onion-Solutions/securityonion#2465] [Unsupported Versions] [answered]
Can anyone please guide how to best use Security Onion 2.0 in AWS and what are the things we can monitor in AWS to monitor security?
Do you recommend using Wazhu set up separately in AWS?
**I am new to this.

**Comment by weslambert** [comment, marked as answer]
It's really up to you.  Some folks choose to use AWS' Traffic Mirroring to aggregate traffic from various instances, while others may choose to use Wazuh/osquery only, or a combination.
